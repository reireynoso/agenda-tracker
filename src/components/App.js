import React from 'react';
import Header from './Header';
import MainComponent from './MainComponent';

export default () => {
    return (
        <main id="app">
            <Header/>
            <MainComponent/>
        </main>
    )
}