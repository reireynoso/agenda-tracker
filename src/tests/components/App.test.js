import React from 'react';
import {BrowserRouter as Router} from 'react-router-dom'
import {Provider} from 'react-redux'
import renderer from 'react-test-renderer'
import App from '../../components/App'
import configureMockStore from 'redux-mock-store'

const createMockStore = configureMockStore()
const component = <Provider store={createMockStore({
    todosReducer: {
        columns: {

        }
    }
})}>   
    <Router>
        <App/>        
    </Router>
</Provider>
test("Render App page", () => {  
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot()
})