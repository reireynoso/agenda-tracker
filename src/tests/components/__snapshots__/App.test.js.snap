// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App page 1`] = `
<main
  id="app"
>
  <header
    id="header"
  >
    <section
      className="align-center"
    >
      <span
        id="header__title"
      >
        Agenda Tracker
      </span>
      <button
        id="header__logout"
        onClick={[Function]}
      >
        Log Out
      </button>
    </section>
  </header>
  <div
    className="align-center"
  >
    <main
      id="main"
    >
      <h1
        id="main__title"
      >
        Agendas
      </h1>
      <div
        className="MuiGrid-root makeStyles-root-1 MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-justify-xs-center"
      />
    </main>
  </div>
</main>
`;
